extensions/azure-keyvault/runtime/pom.xml

```diff
@@ -39,6 +39,19 @@
             <groupId>io.quarkiverse.azureservices</groupId>
             <artifactId>quarkus-azure-core-util</artifactId>
         </dependency>
+
+        <!-- Test Dependencies -->
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-junit5-internal</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <version>3.26.3</version>
```

Comment:
In case other extensions may use the same version of assertj-core, we'd better manage the version with dependencyManagement

Reply:
See https://github.com/quarkiverse/quarkus-azure-services/pull/266/commits/57f60787beb187d26e7b65e7255c3215ac13180f